cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LandingPage.java

```diff
@@ -0,0 +1,146 @@
+package com.skillstorm.PageObjects;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import com.skillstorm.PageObjects.Components.Footer;
+import com.skillstorm.PageObjects.Components.Navbar.LandingNavbar;
+import com.skillstorm.PageObjects.Interfaces.Component;
+
+public class LandingPage extends Page {
+//#region Static fields
+
+    //Naming System
+    public static final String CMP_LANDING_NAVBAR_NAME = "Landing Navbar";    
+    public static final String CMP_FOOTER_NAME = "Footer";
+    public static final String NAME_FEATURES_CONTAINER = "Features";
```

Comment:
Low impact: difference in naming conventions in the same file.

Reply:
fixed